core/jobs.py

```diff
@@ -829,6 +1141,12 @@ def _handle_incoming_event(
 
     @classmethod
     def _get_active_realtime_index(cls):
```

Comment:
An instance has an ID, but this looks like the index of the currently-active layer rather than the ID of an instance, right? If so, both parts of the docstring are wrong and need to be changed.